tests/integrational/fixtures/native_sync/objects_v2/channel_members/get_channel_members_with_pagination.yaml

```diff
@@ -0,0 +1,106 @@
+interactions:
+- request:
+    body: '{"set": [{"uuid": {"id": "test-fix-118-0"}}, {"uuid": {"id": "test-fix-118-1"}},
+      {"uuid": {"id": "test-fix-118-2"}}, {"uuid": {"id": "test-fix-118-3"}}, {"uuid":
+      {"id": "test-fix-118-4"}}, {"uuid": {"id": "test-fix-118-5"}}, {"uuid": {"id":
+      "test-fix-118-6"}}, {"uuid": {"id": "test-fix-118-7"}}, {"uuid": {"id": "test-fix-118-8"}},
+      {"uuid": {"id": "test-fix-118-9"}}, {"uuid": {"id": "test-fix-118-10"}}, {"uuid":
+      {"id": "test-fix-118-11"}}, {"uuid": {"id": "test-fix-118-12"}}, {"uuid": {"id":
+      "test-fix-118-13"}}, {"uuid": {"id": "test-fix-118-14"}}], "delete": []}'
+    headers:
+      Accept:
+      - '*/*'
+      Accept-Encoding:
+      - gzip, deflate
+      Connection:
+      - keep-alive
+      Content-Length:
+      - '568'
+      User-Agent:
+      - PubNub-Python/6.3.0
+    method: PATCH
+    uri: https://ps.pndsn.com/v2/objects/sub-c-33f55052-190b-11e6-bfbc-02ee2ddab7fe/channels/somechannelid/uuids
```

Comment:
may need to review if real keys should be used here

Reply:
That was also my thought but to be honest this is "normal" in those VCR Cassettes. You may see for example [this file](https://github.com/pubnub/python/blob/master/tests/integrational/fixtures/native_sync/objects_v2/channel_members/get_channel_members.yaml). I believe we can ditch them, but it may require regex replacing some portion of tests.

Reply:
definitely we need to ditch real keys in public repo due to potential misuse of keys. If it takes time, perhaps, it can be taken as next task to address it.